lib/shopify_api/conn_helpers.ex

```diff
@@ -0,0 +1,114 @@
+defmodule ShopifyAPI.ConnHelpers do
+  @moduledoc false
+  require Logger
+
+  alias Plug.Conn
+  alias ShopifyAPI.{App, AppServer, AuthTokenServer, Security, Shop, ShopServer}
+
+  @shopify_shop_header "x-shopify-shop-domain"
+  @shopify_topics_header "x-shopify-topic"
+  @shopify_hmac_header "x-shopify-hmac-sha256"
+
+  @doc false
+  def fetch_shopify_app(conn), do: fetch_shopify_app(conn, nil)
```

Comment:
Perhaps better kept as a single arity 2 function, with `app_name` just having a default value of `nil`?